frida/core.py

```diff
@@ -254,7 +254,11 @@ def disable_debugger(self) -> None:
         self._impl.disable_debugger()
 
     @overload
-    def on(self, signal: Literal["message"], callback: Callable[[Mapping[Any, Any], Any], Any]) -> None:
+    def on(self, signal: Literal["destroyed"], callback: Callable[[], None]) -> None:
```

Comment:
If the return value of a callback is ignored I think it is better to have it has `Any` instead of `None`. 
```suggestion
    def on(self, signal: Literal["destroyed"], callback: Callable[[], Any]) -> None:
```